pkgs/development/tools/web-ext/default.nix

```diff
@@ -16,6 +16,8 @@ buildNpmPackage rec {
 
   npmDepsHash = "sha256-KPBKUjCxva11w/E+Qhlx+1vikpCL7Hr9MiKenYHEVSU=";
 
+  preBuild = "export NODE_ENV=production";
```

Comment:
can you add a comment why this is necessary, this is mainly because there are a lot of `buildNpmPackage` that don't require this